packages/grafana-ui/src/components/Select/SelectBase.tsx

```diff
@@ -312,9 +312,6 @@ export function SelectBase<T>({
         }}
         styles={{
           ...resetSelectStyles(),
-          menuPortal: () => ({
-            zIndex: theme.zIndex.dropdown,
-          }),
```

Comment:
I think this will break other stuff, without this the menu portal will use the wrong zIndex. 

What z-index will menuPortal use by default? If not set here?

Reply:
https://github.com/grafana/grafana/pull/23786

Reply:
1, but I think I have a better solution incoming.